Makefile

```diff
@@ -1,7 +1,7 @@
 
 # Image URL to use all building/pushing image targets
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.23
+ENVTEST_K8S_VERSION = 1.27
```

Comment:
right now, we are promising the compatibility of 1.19 ~ 1.25, will this version too high for the compatibility?

Reply:
this modification is by mistake